server/pbenchinacan/container-build.sh

```diff
@@ -1,24 +1,29 @@
-#!/usr/bin/env bash
+#!/usr/bin/env bash -x
 #
-# This script builds a stand-alone Pbench Server container with just the base
-# software installed.
+# This script builds a container which, when run, starts a Pbench Server.
+#
+# Note:  successfully running this script required adding CAP_SETFCAP (for the
+#        installation of httpd) to the list of default capabilities in
```

Comment:
Where are we using `httpd` now?

Reply:
I'm guessing that we're not.  Is there any reason (now) why I should not remove the whole `Note` starting at line 5?